x-pack/plugins/monitoring/public/components/license/index.js

```diff
@@ -53,21 +59,29 @@ const LicenseUpdateInfoForRemote = ({ isPrimaryCluster }) => {
 export function License(props) {
   const { status, type, isExpired, expiryDate } = props;
   return (
-    <EuiPage className="licenseManagement">
+    <EuiPage>
       <EuiPageBody>
-        <div className="licManagement__contain">
-          <LicenseStatus
-            isExpired={isExpired}
-            status={status}
-            type={type}
-            expiryDate={expiryDate}
-          />
-
-          <EuiSpacer />
+        <LicenseStatus
+          isExpired={isExpired}
+          status={status}
+          type={type}
+          expiryDate={expiryDate}
+        />
+        <EuiSpacer />
 
-          <LicenseUpdateInfoForPrimary {...props} />
-          <LicenseUpdateInfoForRemote {...props} />
-        </div>
+        <LicenseUpdateInfoForPrimary {...props} />
+        <LicenseUpdateInfoForRemote {...props} />
+        <EuiSpacer />
+        <EuiText size="m">
```

Comment:
EuiText has a `textAlign` prop directly on it, so you don't need to nest the `EuiTextAlign` component.